service/iot-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/service/RegistryManagerTest.java

```diff
@@ -675,57 +675,6 @@ public void updateDevice_good_case() throws Exception
         new VerificationsInOrder()
         {
             {
-                device.setForceUpdate(false);
-                mockHttpRequest.setHeaderField("If-Match", "*");
-            }
-        };
-    }
-
-    // Tests_SRS_SERVICE_SDK_JAVA_REGISTRYMANAGER_12_034: [The function shall throw IllegalArgumentException if the input device is null]
-    // Assert
-    @Test (expected = IllegalArgumentException.class)
-    public void updateDeviceForce_input_null() throws Exception
```

Comment:
Removing these since we have similar tests for API not taking forceUpdate parameter.